src/tests/tests/background.js

```diff
@@ -119,4 +124,22 @@
     }
   });
 
+  QUnit.test("DNT checking obeys user setting", (assert) => {
+    const NUM_TESTS = 5;
+
+    let done = assert.async(NUM_TESTS);
+
+    assert.expect(NUM_TESTS);
+
+    badger.isCheckingDNTPolicyEnabled = () => false;
```

Comment:
K. in the mean time I'll move the set and unset bits to this test, and mark it with a TODO